docs/interop-ksp.md

```diff
@@ -0,0 +1,126 @@
+KotlinPoet-ksp
+==============
+
+`interop:ksp` is an interop API for converting
+[Kotlin Symbol Processing](https://github.com/google/ksp) (KSP) types to KotlinPoet types and
```

Comment:
nit: I like the style of moving all of the referenced links to the bottom of the document and using short refs, makes markdown easier to read:
```
[Kotlin Symbol Processing][ksp]
...

 [ksp]: https://github.com/google/ksp
```

Reply:
Does that still work in mkdocs?

Reply:
Yes!

Reply:
4ec59a9